common/model-views.cpp

```diff
@@ -960,6 +960,8 @@ namespace rs2
         depth_decoder(std::make_shared<rs2::depth_huffman_decoder>()),
         viewer(viewer),
         detected_objects(device_detected_objects)
+    {}
+    void subdevice_model::init_device_threads(std::string& error_message)
```

Comment:
Not sure this is needed

Reply:
if the code stays inside the constructor it causes error when creating weak pointer to subdevices, because it creates the pointer while still in constructor means that subdevice still not constructed yet but we already have a pointer to it --> crash